packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {ReactHTML, ReactElement} from 'react'
+import clsx from 'clsx'
+import {Icon, IconProps} from '@primer/octicons-react'
+import type {BaseProps} from '../component-helpers'
+import {Heading, Text, Link, HeadingProps, TextProps, LinkProps} from '../'
+
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/bento/colors-with-modes.css'
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/bento/base.css'
+import styles from './Bento.module.css'
+
+export type Size = 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge'
+export type ColumnIndex = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12
+
+type ResponsiveColumnIndex = Partial<Record<Size, ColumnIndex>> | ColumnIndex
+type ResponsiveRowIndex = Partial<Record<Size, number>> | number
+
+type Flow = 'row' | 'column'
+type ResponsiveFlow = Partial<Record<Size, Flow>> | Flow
+
+type Align = 'start' | 'center' | 'end'
+type ResponsiveAlign = Partial<Record<Size, Align>> | Align
+
+type BentoProps = React.HTMLAttributes<HTMLDivElement> & BaseProps<HTMLDivElement>
+
+const Root = ({className, ...rest}: BentoProps) => {
+  return <section className={clsx(styles.Bento, className)} {...rest}></section>
+}
+
+type BentoItemProps = {
+  columnStart?: ResponsiveColumnIndex
+  columnSpan?: ResponsiveColumnIndex
+  rowStart?: ResponsiveRowIndex
+  rowSpan?: ResponsiveRowIndex
```

Comment:
Intellisense is a little misleading here IMO, as the value isn't just a responsive value as the name suggests but a union of responsive _and_ non-responsive values (`Partial<Record<Size, Align>> | Align`), which this value corresponds to. Can we reference the union here instead? We [do the same in Grid](https://github.com/primer/brand/blob/04790926c578c535cd39b410441ca8252180b2d8/packages/react/src/Grid/Grid.tsx#L91), for example.

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/390/commits/1f0c4ef28849a35edff17f5c182554060fbc5167